web/views/sleep_tracker_view.ex

```diff
@@ -22,10 +22,19 @@ defmodule Healthlocker.SleepTrackerView do
     if Kernel.length(past_week) == 0 do
       0
     else
-      total_slept / Kernel.length(past_week)
+      format_average_sleep(total_slept / Kernel.length(past_week))
     end
   end
 
+  # > format_average_sleep(7.375)
+  # > "7h 23m"
```

Comment:
comment does not reflect the format returned. should be `7h 23mins`
also, including this formatting function here means it doesn't have any tests. 🤔